src/styles.css

```diff
@@ -5,12 +5,28 @@
   font-family: "Roboto", sans-serif;
 }
 
-body {
+/* body {
   background: #f1e0ff;
-}
+} */
```

Comment:
Is there a reason we want to remove this? If we are commenting it out can we just remove it completely instead?